make.sh

```diff
@@ -73,14 +74,19 @@ rule() {
       # gprof & ocamlprof (run also generates ocamlprof.dump). use: ocamlprof src/goblint.ml
       ocb -ocamlopt ocamloptp $TARGET.p.native &&
       cp _build/$TARGET.p.native goblint
-    # ;; docs)
-    #   rm -rf doc;
-    #   ls src/**/*.ml | egrep -v $EXCLUDE  | sed 's/.*\/\(.*\)\.ml/\1/' > doclist.odocl;
-    #   ocb -ocamldoc ocamldoc -docflags -charset,utf-8,-colorize-code,-keep-code doclist.docdir/index.html;
-    #   rm doclist.odocl;
-    #   ln -sf _build/doclist.docdir doc
-    # ;; tag*)
-    #   otags -vi `find src/ -iregex [^.]*\.mli?`
+    ;; docs)
+      rm -rf doc;
+      ls src/**/*.ml | egrep -v $EXCLUDE  | sed 's/.*\/\(.*\)\.ml/\1/' > doclist.odocl;
+      ocb -ocamldoc ocamldoc -docflags -charset,utf-8,-colorize-code,-keep-code doclist.docdir/index.html;
+      rm doclist.odocl;
+      ln -sf _build/doclist.docdir doc
```

Comment:
What's the reason for reintroducing this? It's using the old ocamlbuild instead of dune, which can build documentation with a single command: https://dune.readthedocs.io/en/stable/documentation.html.